README.md

```diff
@@ -378,11 +378,21 @@ mvn dependency:copy-dependencies
 
 * Configure Kafka Connectors with following properties
 
+When configuring the worker or connectors in Kafka Connect, refer to the string constant properties of the [AWSSchemaRegistryConstants](https://github.com/awslabs/aws-glue-schema-registry/blob/master/common/src/main/java/com/amazonaws/services/schemaregistry/utils/AWSSchemaRegistryConstants.java#L20) class.
+
+Examples:
+
+AWSSchemaRegistryConstants.AWS_REGION = "region"
```

Comment:
IMO, this might confuse users as it's followed by code. We don't need the Examples as the link above points to code.

Reply:
Good call, that makes sense to me. The examples have been removed.